components/footer/footer-group.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import useTheme from '../use-theme'
+import { useFooterContext } from './footer-context'
+
+interface Props {
+  title: string | React.ReactNode
+}
+
+const defaultProps = {
+  title: '',
+}
+
+type NativeAttrs = Omit<React.HTMLAttributes<any>, keyof Props>
+export type FooterGroupProps = Props & typeof defaultProps & NativeAttrs
+
+const FooterGroup: React.FC<React.PropsWithChildren<FooterGroupProps>> = ({
+  title,
+  children,
+  ...props
+}) => {
+  const theme = useTheme()
+  const { breakPoint } = useFooterContext()
+  const [expand, setExpand] = React.useState(false)
+  return (
+    <div {...props}>
+      <h3 onClick={() => setExpand(!expand)}>{title}</h3>
+      <ul>{children}</ul>
+
+      <style jsx>{`
+        div{
+          margin-bottom:20px;
+        }
+        h3 {
+          font-weight: 400;
+          font-size: 0.875rem;
+          margin: ${theme.layout.gapHalf} 0;
+        }
+        ul {
+          list-style-type: none !important;
```

Comment:
Do not use `important`, if you want to override the style, please refer to `styled-jsx` for dynamic styles: `resolve`, we have used it in the `Grid` component.